lib/runtime/wazero/instance.go

```diff
@@ -100,15 +108,13 @@ func NewInstanceFromTrie(t trie.Trie, cfg Config) (*Instance, error) {
 	return NewInstance(code, cfg)
 }
 
-// NewInstance instantiates a runtime from raw wasm bytecode
-func NewInstance(code []byte, cfg Config) (instance *Instance, err error) {
-	logger.Info("instantiating a runtime!")
-	logger.Patch(log.SetLevel(cfg.LogLvl), log.SetCallerFunc(true))
-
-	ctx := context.Background()
-	rt := wazero.NewRuntime(ctx)
+func newRuntimeInstance(ctx context.Context,
```

Comment:
I feel like this function name should be changed given it doesn't return an `Instance` anymore.

Reply:
updated: renamed to `newRuntime` since a runtime and associated data are still being created. Let me know if you prefer something else

Reply:
Agreed, maybe change to compiled modules

Reply:
Hm but it returns one runtime and one module (since we don't cache host module anymore). Maybe something like `newRuntimeWithGuestModule`?